portfolio/src/main/webapp/index.html

```diff
@@ -8,14 +8,25 @@
   </head>
   <body>
     <div id="content">
-      <h1>Yenessa's Portfolio</h1>
-      <p>This is my portfolio!! :D</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Ayyy</button>
-      <div id="greeting-container"></div>
-      <p>I'm currently a sophomore studying Computer Engineering at the University of Florida. I'm passionate about the societal impact of technology and the integration between hardware and software.</p>
-      <p>I also loooove cats! Here's one of my favorite cat images. </p>
-      <img src="images/catinbox.JPG"/>
+        <body style="background-color:thistle;">
+        <font face="garamond">
```

Comment:
I'd strongly recommend using CSS rather than font tags. `<font>` isn't really used anymore.

You should also specify a fallback font family in case the font specified isn't on the client's computer. Sans-serif fonts are typically recommended for digital content.

Here's what you might put in your css file.

`body {
  font-family: garamond sans-serif;
  background-color: thistle;
}`

This says: for all elements of type `body` apply the font-family garamond if available. If not available, use the client browser's default sans-serif font. Also apply background color thistle.

Looks like you already have a CSS file called style.css where you can put this.